content/docs/reference/pkg/gcp-native/_index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "gcp-native"
```

Comment:
I think we need to add a user-friendly alias in pulumi/pulumi

Reply:
> I think we need to add a user-friendly alias in pulumi/pulumi

👍🏾 . I discovered a codegen bug which currently makes gcp native python sdk unusable. Will roll that in as part of that fix.

Reply:
This was merged in as part of https://github.com/pulumi/pulumi/pull/6748